tests/integration/golang/mlflow/run/log_parameter_test.go

```diff
@@ -1,3 +1,190 @@
 //go:build integration
 
 package run
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/api"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/api/request"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/fixtures"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type LogParamTestSuite struct {
+	suite.Suite
+	client             *helpers.HttpClient
+	runFixtures        *fixtures.RunFixtures
+	experimentFixtures *fixtures.ExperimentFixtures
+	run                *models.Run
+}
+
+func TestLogParamTestSuite(t *testing.T) {
+	suite.Run(t, new(LogParamTestSuite))
+}
+
+func (s *LogParamTestSuite) SetupTest() {
+	s.client = helpers.NewHttpClient(os.Getenv("SERVICE_BASE_URL"))
+	runFixtures, err := fixtures.NewRunFixtures(os.Getenv("DATABASE_DSN"))
+	assert.Nil(s.T(), err)
+	s.runFixtures = runFixtures
+	expFixtures, err := fixtures.NewExperimentFixtures(os.Getenv("DATABASE_DSN"))
+	assert.Nil(s.T(), err)
+	s.experimentFixtures = expFixtures
+
+	expName := uuid.New().String()
```

Comment:
this is not re used somewhere so you use it inline:
```
Name: uuid.New().String()
```

Reply:
done